src/cascadia/TerminalSettingsModel/TerminalSettingsSerializationHelpers.h

```diff
@@ -168,7 +167,7 @@ struct ::Microsoft::Terminal::Settings::Model::JsonUtils::ConversionTrait<::winr
         }
 
         ::winrt::Windows::UI::Text::FontWeight weight{
-            static_cast<uint16_t>(std::clamp(value, 100u, 990u))
+            static_cast<uint16_t>(std::clamp(value, 1u, 999u))

```

Comment:
I'm sorry, what? 1 is not a valid font weight

Reply:
> I'm sorry, what? 1 is not a valid font weight

Typically goes from 100 to 990 I think - but I believe the font sets its minimum and maximum values?

https://docs.microsoft.com/en-us/typography/opentype/spec/fvar

``
Example
This example is for a hypothetical font with family name “SelawikV” that has two axes of variation, for weight and width. This table summarizes the description of the axes for the font:
``
Axis tag | Minimum value | Default value | Maximum value | Axis name ID
-- | -- | -- | -- | --
'wght' | 300 | 400 | 700 | 256
'wdth' | 62.5 | 100 | 150 | 257

Reply:
But the documentation for [FontWeight](https://docs.microsoft.com/en-us/uwp/api/windows.ui.text.fontweight?view=winrt-19041) says that this is the supported range though. Do we clamp it somewhere else?

Reply:
I backed this out and clamped it to 100 on deserialization. On a round trip it will be "fixed".